examples/ggez/main.rs

```diff
@@ -0,0 +1,130 @@
+mod map;
+
+use ggez::{event::{self, MouseButton}, Context, GameResult, graphics::{self, DrawParam}, input};
+use map::MapHandler;
+use tiled::{Map, FilesystemResourceCache};
+
+fn main() -> GameResult {
+    // init ggez
+    let cb = ggez::ContextBuilder::new("rs-tiled + ggez", "rs-tiled")
+        .window_setup(ggez::conf::WindowSetup::default()
+            .title("rs-tiled + ggez example")
+            .vsync(false)
+        )
+        .window_mode(ggez::conf::WindowMode::default()
+            .dimensions(1000.0, 800.0)
+        )
+        .add_resource_path(""); // add repo root to ggez filesystem (our example map looks for `assets/tilesheet.png`)
```

Comment:
I believe this should be `std::env::var("CARGO_MANIFEST_DIR")` so that the example works in directories within the root one

Reply:
```suggestion
        // add repo root to ggez filesystem (our example map looks for `assets/tilesheet.png`)
        .add_resource_path("std::env::var("CARGO_MANIFEST_DIR").unwrap()");
```